.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Weekly Wednesday 2AM build
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 2 * * 3'
+
+env:
+  MPLBACKEND: "Agg"
+
+# TODO: Use ccache?
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        astropy_version: [4.0, 4.1, dev]
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install
+      run: |
+        sudo apt-get install mlocate
+        python -m pip install scipy matplotlib asv
+        asv machine --machine Actions --os unknown --arch unknown --cpu unknown --ram unknown
+    - name: Test with stable astropy
+      run: |
+        python -m pip install astropy=={{ matrix.astropy_version }}
+        asv run -q
```

Comment:
`You have an error in your yaml syntax on line 36`

Need help from someone with YAML-fu.